src/status_im2/common/standard_auth/keycard/style.cljs

```diff
@@ -0,0 +1,62 @@
+(ns status-im2.common.standard-auth.keycard.style
+  (:require
+    [quo2.foundations.colors :as colors]))
+
+(def container
+  {:flex 1})
+
+(def inner-container
+  {:flex               1
+   :padding-horizontal 20})
+
+(def context-tag
+  {:margin-top     12
+   :margin-bottom  56
+   :flex-direction :row})
+
+(def close-button
+  {:margin-bottom 12})
+
+(def secondary-text
+  {:margin-top    8
+   :margin-bottom 20})
+
+(def try-again-button
+  {:flex               1
+   :padding-horizontal 20})
+
+(def keycard
+  {:margin-bottom 12})
+
+(def divider
+  {:margin-top 20})
+
+(def reset-keycard-button
+  {:align-self
+   :flex-start
+   :margin-top 12})
+
+(defn get-digits-container-style
```

Comment:
Style functions are almost always defined as nouns, like `digits-container` and don't start with a verb like `get-`.

Reply:
@ilmotta Makes sense. Removed the `get`.